eng/common/scripts/Get-GitHubUserName.ps1

```diff
@@ -0,0 +1,69 @@
+<#
+.SYNOPSIS
+Retrieved the github user information using microsoft username or email
+
+.DESCRIPTION
+This script take either a s microsoft username or the alias version of the email and returns the github user information associated with that user.
+
+.PARAMETER IdentityName
+Full official microsoft username of a user.
+
+.PARAMETER IdentityEmail
+The microsoft aliase email of a user.
+
+.EXAMPLE
+PS> ./eng/common/scripts/Get-GitHubUserName.ps1 -IdentityName "Chidozie Ononiwu" -IdentityEmail "chononiw@microsoft.com"
+
+You must provide one of the parameters.
+#>
+param(
+    [string]$IdentityName,
+    [string]$IdentityEmail,
+    [string]$TargetDevOpsVariable,
+    [string]$ToolVersion,
+    [string]$ToolPath = (Join-Path ([System.IO.Path]::GetTempPath()) "identity-converter-tool-path"),
+    [string]$DevOpsFeed
+)
+
+. (Join-Path $PSScriptRoot common.ps1)
+. (Join-Path $PSScriptRoot Helpers DotnetTool-Helpers.ps1)
+
+if (!$IdentityName -and !$IdentityEmail)
+{
+    LogError "You must provide either 'IdentityName' or 'IdentityEmail'"
+    exit(1)
+}
+
+$command = Get-CodeOwnersTool -toolPath $ToolPath -toolName "Azure.Sdk.Tools.IdentityConverter" -toolVersion $ToolVersion `
+-feedUrl $DevOpsFeed -toolCommandName "identity-converter"
```

Comment:
```suggestion
$command = Get-CodeOwnersTool `
                        -toolPath $ToolPath `
                        -toolName "Azure.Sdk.Tools.IdentityConverter" `
                        -toolVersion $ToolVersion `
                        -feedUrl $DevOpsFeed -toolCommandName "identity-converter"
```

Though for some reason the diff whitespace is showing oddly in the github display.